test/unit/core.test.ts

```diff
@@ -65,14 +66,20 @@ describe('getJoiSchema', () => {
             fluffinessIndex: jf.joi.number().min(1).max(5),
         });
 
-        expect(JSON.stringify(getJoiSchema(Cat, jf.joi))).toEqual(JSON.stringify(expectedSchema));
+        expect(stringify(getJoiSchema(Cat, jf.joi))).toEqual(stringify(expectedSchema));
```

Comment:
New joi object has circular references so had to use a stringify library that doesn't explode on circular refs.